cpp/include/cudf/strings/capitalize.hpp

```diff
@@ -30,21 +30,31 @@ namespace strings {
 /**
  * @brief Returns a column of capitalized strings.
  *
- * Any null string entries return corresponding null output column entries.
+ * If the `delimiters` is an empty string, then only the first character of each
+ * row is capitalized. Otherwise, a non-delimiter character is capitalized after
+ * any delimiter character is found.
  *
  * @code{.pseudo}
  * Example:
- * input = ["tesT1", "a Test", "Another Test"];
+ * input = ["tesT1", "a Test", "Another Test", "a\tb"];
  * output = capitalize(input)
- * output is ["Test1", "A test", "Another test"]
+ * output is ["Test1", "A test", "Another test", "A\tB"]
```

Comment:
I think this output is not quite right since it's supposed to capitalize only the first character when no delimiters are specified.
```suggestion
 * output is ["Test1", "A test", "Another test", "A\tb"]
```

Reply:
Yes, thanks for catching that.